### PR TITLE
Show 10 events per page for admin events

### LINKED ISSUE #453: Per page 10 events for admin/events
@denysmedynskyi we have available human resources?

### PR DESCRIPTION
**Pull request by Sayok**
Fixed #453 

### Description

Added more convenient pagination for admin/events page 10 entries per page.

### How to test instructions

1. go to /admin/events page verify that there are 25 entries per page
2. switch to local branch "453-show-10-events-per-page-for-admin-events"
4. go to /admin/events page verify that there are 10 entries per page

### Pre-merge checklist
 
- [+ ] The PR relates to a single subject with a clear title and description
- [ +] Verify that feature branch is up-to-date with `development` (if not - rebase it). 

### Screenshots

| Before                                      | After                                       |
| ------------------------------------------- | ------------------------------------------- |
| 
before_25_per_page | after_25_per_page
 |

### CODE REVIEW

**Review by sveredyuk — DISMISSED** [review]

File: spec/features/admin/events/read_spec.rb [hunk 1]
```diff
@@ -16,4 +16,22 @@
...
+
+  context 'do not paginate less than 10 records on the page' do
+    it 'does not have second page' do
+      expect(page).not_to have_xpath("//*[@class='ui pagination menu']//a[text()='2']")
+    end
+  end
+
+  context 'paginate more than 10 records on the page' do
+    before do
+      11.times { FactoryGirl.create(:event) }
```
Comment (sveredyuk) [inline spec/features/admin/events/read_spec.rb:28]: What about some mock ?

Comment (denys-medynskyi) [reply spec/features/admin/events/read_spec.rb:28]: I disagree. It is an integration test - it should use the real stuff.

File: app/controllers/admin/events_controller.rb [hunk 2]
```diff
@@ -31,7 +31,7 @@ def event
     end
 
     def events
-      @events ||= ::Event::List.(events: search_against(Event), page: params[:page])
+      @events ||= ::Event::List.(events: search_against(Event), page: params[:page]).per(10)
```
Comment (denys-medynskyi) [inline app/controllers/admin/events_controller.rb]: Hardcoded values are not good (`10`)
This `10` value could be extracted to settings.yml, but
@sveredyuk wrote cool engine to use settings throughout the app - [ez-settings](https://github.com/ez-engines/ez-settings)
I think we need to use it to be consistent.
@sveredyuk do you agree?

**Review by sveredyuk — DISMISSED** [review]
WOW!

File: spec/features/admin/events/read_spec.rb (hunk 1 above)
Comment (sveredyuk) [reply spec/features/admin/events/read_spec.rb:28]: Ok (

File: .rubocop.yml [hunk 3]
```diff
@@ -1,1156 +0,0 @@
-AllCops:
-  DisabledByDefault: true
-
```
Comment (sveredyuk) [inline .rubocop.yml]: Hey! Why you deleted rubocop file?

**Review by floor114 — COMMENTED** [review]
Why did you remove `.rubocop.yml`?

**Review by sveredyuk — APPROVED** [review]

**Comment by floor114** [comment]
@Sayok Could you squash all commits into one?

**Comment by denys-medynskyi** [comment]
@floor114 they will be squashed during the merge via github admin. Do you agree?

**Comment by floor114** [comment]
@denysmedynskyi, course)

**Review by floor114 — APPROVED** [review]

**Pull request merged**